qml/FileListPage.qml

```diff
@@ -292,8 +293,14 @@ Page {
                         //: Share a file
                         //% "Share"
                         text: qsTrId("sailfish-office-la-share")
-                        onClicked: pageStack.animatorPush("Sailfish.TransferEngine.SharePage",
-                                                          { serviceFilter: ["e-mail"], mimeType: model.fileMimeType, source: model.filePath })
+                        onClicked: {
+                            shareAction.fileResources = [model.filePath]
+                            shareAction.trigger()
+                        }
+                        ShareAction {
+                            id: shareAction
+                            capabilitiesFilter: model.fileMimeType
```

Comment:
It previously used a serviceFilter "e-mail" as well as a mime type filter.  Was this change intentional?  Similarly for the other cases above.

Reply:
Yes, it's okay to remove that now as that feature was only used to configure the in-app "add account" feature, but the user will now be sent to the Settings app so it's no longer necessary.